gui-tk/wiring.tcl

```diff
@@ -658,7 +658,13 @@ proc contextMenu {x y X Y} {
             }
             .wiring.context add command -label "Flip" -command "$item.flip; flip_default"
             if {[$item.type]=="parameter"} {
-                .wiring.context add command -label "Import CSV" -command CSVImportDialog
+                .wiring.context add command -label "Import CSV" -command {CSVImportDialog {}}
+                .wiring.context add command -label "Import CSV from web" -command {
+					textEntryPopup .loadWebUrl {} {CSVImportDialog [.loadWebUrl.entry get]}
+                .loadWebUrl.entry configure -takefocus 1					
+				wm title .loadWebUrl "Insert URL:"		
+                wm geometry .loadWebUrl "+[winfo pointerx .]+[winfo pointery .]"						 				
```

Comment:
Indent the code to make this clear this is part of the -command script.  I found this misleading initially.

Reply:
Done.